doc/usage.txt

```diff
@@ -1,23 +1,27 @@
 OpenFastTrace
 
 Usage:
-  oft trace [option ...] [<input file or directory> ...]
+  oft command [option ...] [<input file or directory> ...]
 
-Commands
+Commands:
   trace                        Create a trace document
   convert                      Convert to a different requirements format
 
 Tracing options:
   -o, --output-format          Report format, one of "plain"
                                Defaults to "plain"
+  -v, --verbosity-level        Set how verbose the output is. Ranges from
+                               "quiet" to "all".
```

Comment:
Are there other possible values?
* If yes: list all.
* If no: write "possible values: ..." or "one of ..."